README.md

```diff
@@ -1 +1,99 @@
-# web-design-curriculum
\ No newline at end of file
+# Web Design
+> Web design refers to the process of creating the visual appearance and layout of websites. It involves a combination of graphic design, user interface (UI) design, and user experience (UX) design principles to produce a website that is visually appealing, functional, and easy to use
+
+## Prerequisites
+No prior designing experience is required to start this course. However, having a basic understanding of computer operations and familiarity with using a computer is recommended.
+
+## Installation and Setup
+To use this curriculum and practice it, you will need to install the following:
+
+1. [Figma](https://code.visualstudio.com/):  A collaborative interface design tool that enables real-time collaboration, prototyping, and vector-based graphics creation for teams.
+
+## 🤔 How to Use This Curriculum
+This curriculum is designed to be self-paced, and you can learn at your own speed. It is also designed to be hands-on, and you will be building real-world projects as you learn. The curriculum is divided into different sections, and each section covers a specific topic. You can start with the first section and work your way through the curriculum. Each section contains a list of resources, including articles, tutorials, and exercises, that you can use to learn the topic. The videos for this curriculum will be provided offline, allowing you to access the content without an internet connection. However, exercises and project submissions will be done online.
+
+## 📝 Introduction
+Web design refers to the process of creating the visual appearance and layout of websites. It involves a combination of graphic design, user interface (UI) design, and user experience (UX) design principles to produce a website that is visually appealing, functional, and easy to use.
```

Comment:
This is the same as the first description text at the beginning of the readme so change it a little bit.